src/vs/workbench/parts/extensions/electron-browser/extensions.contribution.ts

```diff
@@ -217,6 +217,11 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 				type: 'boolean',
 				description: localize('extensionsShowRecommendationsOnlyOnDemand', "If set to true, recommendations will not be fetched or shown unless specifically requested by the user."),
 				default: false
+			},
+			'extensions.closeExtensionDetailsOnViewChange': {
+				type: 'boolean',
+				description: localize('extensionsCloseExtensionDetailsOnViewChange', "If set to true, extension information tabs will be automatically closed upon navigating away from the Extensions View."),
+				default: false
```

Comment:
@JacksonKearl The default value for this setting should be `true` as per https://github.com/Microsoft/vscode/issues/51005#issuecomment-395015087

Reply:
Right now the default is "It is not the case that we should `closeExtensionDetailsOnViewChange`", which is as was discussed. Would it be better if we negated the name so that the default was "It is the case that we should `keepExtensionDetailsOpenOnViewChange`"?